dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/resources/bootstrap.yaml

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
Since we have a new `bootstrap.yaml` configuration for all components, do we need to update https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/architecture/configuration.md and add some related instructions?

Reply:
> Since we have a new `bootstrap.yaml` configuration file for all components, do we need to update https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/architecture/configuration.md and add some related instructions?

Thank you, I've updated those two docs